packages/blade/src/components/Checkbox/checkboxTokens.ts

```diff
@@ -5,18 +5,36 @@ import { size } from '~tokens/global';
 
 const checkboxSizes = {
   group: {
-    gap: {
-      small: {
-        mobile: 'spacing.3',
-        desktop: 'spacing.2',
+    vertical: {
```

Comment:
spacing seems same in both cases so do we need separate vertical and horizontal keys here?

Also is it expected to change the spacings in vertical scenarios as well compared to what it was earlier?

Reply:
yes , inspected values in design